onnxmltools/utils/main.py

```diff
@@ -81,11 +83,17 @@ def save_text(model, file_path):
         from onnxmltools.utils import save_text
         save_text(onnx_model,"SqueezeNet.json")
     """
+    if not isinstance(model, onnx_proto.ModelProto):
+        raise ValueError("Model is not a valid ONNX model.")
     try:
         with open(file_path, "w") as f:
             f.write(str(model))
     except IOError:
-        print("Could not save file")
+        print("Unable to write file to path '{0}', check if you have permissions.".format(file_path))
+        return
+    except:
+        print("Failed trying to save file '{0}'.".format(file_path))
+        return
```

Comment:
Is this exception catching really necessary?

If something goes wrong e.g. `file_path` does not exist or `str(model)` raises an exception it's better IMO to get the actual exception than an obscure `Failed trying to save file '<file_name>'` that doesn't provide any useful information for bug reports..

Reply:
Thanks for the suggestions. I agree that it makes more sense not to catch a general except. See latest update!

Reply:
@wenbingl, is there a reason why the team chose to use print statements instead of exceptions?